challenge-06/challenge-06.js

```diff
@@ -32,20 +35,50 @@ Crie uma função chamada `showTeamPosition` com as seguintes características:
     - Se não houver time para a posição passada, deve retornar a mensagem:
     "Não temos a informação do time que está nessa posição."
 */
-// ?
+
+function showTeamPosition(position){
+    var teamPositions = "O time que está em " + position + "°" + " lugar é o ";
+    switch(position){
+        case 1 :
+        return teamPositions + teams[position -1] + ".";
+        break;
+        case 2 :
+        return teamPositions + teams[position -1] + ".";
+        break;
+        case 3 :
+        return teamPositions + teams[position -1] + ".";
+        break;
+        case 4 :
+        return teamPositions + teams[position -1] + ".";
+        break;
+        case 5 :
+        return teamPositions + teams[position -1] + ".";
+        break;
+        default :
+        return "Não temos a informação do time que está nessa posição.";
+    }
+
+}
 
 /*
 Escolha 4 times do campeonato selecionado e mostre a posição dele, usando a
 função acima. Entre esses 4, adicione 1 que não esteja entre os 5 primeiros.
 */
-// ?
+
+console.log(showTeamPosition(1)); // O time que está em 1° lugar é o flamengo.
+console.log(showTeamPosition(3)); //O time que está em 3° lugar é o fluminence.
+console.log(showTeamPosition(5)); // O time que está em 5° lugar é o botafogo.
+console.log(showTeamPosition(7)); //Não temos a informação do time que está nessa posição.
 
 /*
 Mostre os números de 20 a 30 no console (inclusive o 30), usando a estrutura de
 repetição "while".
 */
-// ?
 
+var contagem = 19;
```

Comment:
Deixando a contagem começar em `19` você acaba gerando um número que não faz muito sentido, pois queremos ver os números de `20` a `30`, inclusive esses dois.

A melhor abordagem seria somente alterar o sinal de `++` dentro do `while` para **depois** da variável. Com isso, você faz o pós incremento, mostrando então o número 20 :)

Assim, esse código vai funcionar como esperamos, mas o ideal é que o seu código fique o mais legível e claro possível para qualquer pessoa que for ler. E nesses casos, muitas vezes é necessário mudar a nossa abordagem :)

Conseguiu compreender? Se ficou alguma dúvida, me avise :D